x-pack/legacy/plugins/maps/public/actions/map_actions.d.ts

```diff
@@ -24,3 +26,45 @@ export function updateSourceProp(
   value: unknown,
   newLayerType?: LAYER_TYPE
 ): void;
+
+export interface MapCenter {
+  lat: string;
+  lon: string;
+  zoom: unknown;
+}
+
+export function setGotoWithCenter(config: MapCenter): AnyAction;
+
+export function replaceLayerList(layerList: unknown): AnyAction;
+
+export interface QueryGroup {
+  filters: Filter[];
+  query?: Query;
+  timeFilters: unknown;
+  refresh: unknown;
+}
+
+export function setQuery(query: QueryGroup): AnyAction;
+
+export interface RefreshConfig {
+  isPaused: boolean;
+  interval: unknown;
+}
+
+export function setRefreshConfig(config: RefreshConfig): AnyAction;
+
+export function disableScrollZoom(): AnyAction;
+
+export function disableInteractive(disable: boolean): AnyAction;
```

Comment:
disableInteractive, disableTooltipControl, hideToolbarOverlay, hideLayerControl, and hideViewControl do not take any parameters. It looks like they are passed parameters in MapEmbeddable but the parameters are not used in the actions.

https://github.com/elastic/kibana/blob/7.7/x-pack/legacy/plugins/maps/public/actions/map_actions.js#L910